islands/GlobalSearch.tsx

```diff
@@ -78,13 +78,90 @@ const client = algoliasearch("QFPCRZC6WX", "2ed789b2981acd210267b27f03ab47da", {
   requester,
 });
 
+const defaultResult: Results = {
```

Comment:
Let's comment how these data are created and can be updated in the future above this line